TensorFlowV2_CNN_MNIST/mnist.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import tensorflow_datasets as tfds
+import tensorflow as tf
+
+BUFFER_SIZE = 10000
+BATCH_SIZE = 64
```

Comment:
We should provide a parametrized script for testing and comparing combinations of training settings provided as parameters. Batch size should be definitely one of them.

Potentially, another one could be the chosen strategy, if it makes sense to compare them.

See parameters in https://github.com/tensorflow/benchmarks/tree/master/scripts/tf_cnn_benchmarks.

Reply:
BTW, instead of many files with redundant model definitions, we should have one parametrized script, or at least a few scripts but sharing the same model code.

Reply:
1. scripts are now parametrized
2. model definition and dataset preparation moved to utils.py file
3. I have left separate train scripts,. In my opinion it helps to clearly see adjustments in code, which are required for different strategies.

Reply:
1. OK but the parameters should be listed in README
2. OK, thx
3. OK, thx

Reply:
Done.